web/client/epics/wfsquery.js

```diff
@@ -122,8 +124,16 @@ const getWFSFilterData = (filterObj) => {
 
 const getWFSFeature = (searchUrl, filterObj) => {
     const data = getWFSFilterData(filterObj);
+
+    const urlParsedObj = Url.parse(searchUrl, true);
+    const parsedUrl = urlParsedObj.protocol + '//' + urlParsedObj.host + urlParsedObj.pathname + '?';
+    let params = isObject(urlParsedObj.query) ? urlParsedObj.query : {};
+    params.service = 'WFS';
+    params.outputFormat = 'json';
+    const paramsString = querystring.stringify(params);
```

Comment:
You don't need other libraries. You can do all with [Url.format](https://www.npmjs.com/package/url#urlformaturlobj).
Create a new object from urlParsedObj modifying the query object and call Url.format with the new object.